src/modules/facility-fixed-fee/facility-fixed-fee.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { DateOnlyString } from '@ukef/helpers';
+
+export interface FacilityFixedFee {
+  facilityIdentifier: string;
+  portfolioIdentifier: string;
+  amount: number;
+  effectiveDate: DateOnlyString;
+  expirationDate: DateOnlyString;
+  nextDueDate: DateOnlyString;
+  nextAccrueToDate: DateOnlyString;
+  period: string;
+  description: string;
+  currency: string;
+  lenderTypeCode: string;
+  incomeClassCode: string;
+  spreadToInvestorsIndicator: boolean;
+}
```

Comment:
If the one-to-one mapping in the controller is removed (see comment above), this interface will no longer be needed.